desktop/cmp/grid/editors/HoistInlineEditorModel.js

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
This could also be called "CellEditorModel".  In any case, seems like filename has an extra ''Hoist"